service-registry/src/main/java/io/servicecomb/serviceregistry/registry/AbstractServiceRegistry.java

```diff
@@ -225,6 +226,39 @@ public boolean unregisterInstance() {
     return instances;
   }
 
+  public MicroserviceInstances findServiceInstances(String appId, String serviceName,
```

Comment:
io.servicecomb.serviceregistry.registry.AbstractServiceRegistry.findServiceInstance(String, String, String)
can invoke this one, no need to keep two copy.

Reply:
done